distribution_circuits_sale/models/sale.py

```diff
@@ -127,8 +127,14 @@ def _compute_amount_due(self):
             
             for invoice in open_invoices:
                 amount_total += invoice.residual
+            
+            # only not invoiced lines of orders are not taken into account in customer credit
+            orders = order_obj.search([('partner_id','=',partner.id),('state','in',['sent','sale']),('invoice_status','!=','invoiced')])
+            
+            for order in orders:
+                for order_line in order.order_line:
+                    if order_line.invoice_status != 'invoced':
+                        amount_total += order_line.price_total

```

Comment:
What do you think about this change?

Reply:
I curious to know to which use cases this code apply?

Reply:
@francoisdelarbre tag me on your comment next time otherwise I won't be notified of your comment.

Reply:
@houssine78, It's when we do not deliver all the ordered products at the same time. The product not delivered yet are not invoiced. At Hesbicoop, It happens quite often with the meat that has to be ordered several weeks before it is actually delivered.

Reply:
@francoisdelarbre line 136 : shouldn't it be 'invoiced' instead?

Reply:
@nthnnl indeed @waligaliIT ok I see your point. As I've implemented to have two time frames in parallel this doesn't apply anymore.